libs/common/src/sql-db-entities/utilisation-report/utilisation-report.entity.ts

```diff
@@ -82,10 +91,18 @@ export class UtilisationReportEntity extends AuditableBaseEntity {
     this.updateLastUpdatedBy(requestSource);
   }
 
+  /**
+   * Updates the report with fee records
+   * @param param - The parameters to update the report with
+   */
   public updateWithFeeRecords({ feeRecords }: UpdateWithFeeRecordsParams): void {
     this.feeRecords = feeRecords;
   }
 
+  /**
+   * Updates the report with a supplied status
+   * @param param - The parameters to update the report with
```

Comment:
Can the parameters names please be reflected, I would encourage adding `@returns` as well if below snippet can be used as a reference please that will be great. Specifying types in TSDocs is optional.

```typescript
/**
 * Updates the report with a supplied status.
 *
 * @param {UpdateWithStatusParams} params - An object containing the status and request source.
 * @param {string} params.status - The new status to update.
 * @param {string} params.requestSource - The source of the update request.
 * @returns {void} - This method does not return a value.
 */
```

Reply:
Thanks for adding properties to the docstring.
TSDocs reference on `@returns` https://tsdoc.org/pages/tags/returns/, happy to approve on this instance as DocStrings needs to be unified under an established tech-debt ticket.

Reply:
That reference highlights that `@returns` documents a return _value_, where `void` is designed to document the absence or a return value, hence the distinction in this case